### ISSUE TITLE
SQL Language Help

### ISSUE THREAD

**Issue by jeff-bowles** [dbeaver/dbeaver#23471]
The existing context driven help is not what I need most of the time.  There is an existing ticket " #Context based help/wiki navigation. #18416" with a good suggestion for improving that.

However, I'd like to request an alternative help facility that is geared toward the SQL language itself.  Ideally, I would put my cursor on a keyword (like **COALESCE**, for a trivial example) and press a key (Alt-F1, for example) and be taken to a resource that explains the syntax of the keyword in the SQL variant of the current connection.

I imagine needing a way to tie the specific connection's SQL variant to a specific resource in the connection properties, perhaps.  Maybe even with tokens being passed.  So I would configure my Yellowbrick connections to use https://docs.yellowbrick.com/5.2.28/ybd_sqlref/TOKEN.html  to take me to the Yellowbrick documentation online, and the token (**COALESCE**) would be merged in to the URL to take me to the specific topic.

Some SQL variants may not have an online help facility.  In those cases, maybe the connection-specific resource could be a PDF file, with a search on the keyword?  Again, for example, I could configure my connection to use "acrobat.exe /A search=TOKEN c:\mydocument.pdf" as a help resource.

Currently, I either keep a browser window open to the language-specific documentation I need or keep a PDF open for the several SQL flavors I work with.  This works pretty well, but the ability to just hit a key and go to the right documentation would be really nice to have.

Thanks!

Jeff Bowles

**Comment by E1izabeth** [comment]
Similar issue #10237

**Comment by jeff-bowles** [comment]
You’re right, the issues are almost exactly the same.  I searched (but did not find #19237) the tickets before creating my own.  Sorry about the duplication.
